bin/build_auth_release.yaml

```diff
@@ -0,0 +1,8 @@
+# Used for automated builds triggered on Git tags pushed matching the regex:
+#   \d+\.\d+\.\d+
+
+steps:
+- name: 'gcr.io/$PROJECT_ID/istio-builder'
+  args: [ 'bash', './bin/push-docker.sh', '-h', "gcr.io/$PROJECT_ID", '-t', "$TAG_NAME" ]
```

Comment:
It would be good to create a release.sh, but if not use gcr.io/istio-io,docker.io/istio for -h